cilium/cmd/bpf_proxy_list.go

```diff
@@ -32,16 +28,20 @@ const (
 	destinationTitle = "DESTINATION"
```

Comment:
Not really relevant to your PR, but I think it'd be a bit clearer if these two were `PROXY CONNECTION` and `ORIGINAL DESTINATION`.

Reply:
I'm changing this in another PR anyway as I have extended it to include identity information and lifetime